labelparse.py

```diff
@@ -13,21 +13,17 @@
                   'terraform-newrelic-apm', 'newrelic-cloudformation', 'newrelic-cli', 'newrelic-kubernetes-operator', 'newrelic-client-go', 'developer-toolkit']
 
 
-def print_labels(issue):
-    '''Loops through repo data'''
-
-    if 'labels' in issue and issue['labels'] != []:
-        for label in issue['labels']:
-            print('Label: ' + label['name'])
-
-
 def print_issue(issue):
     '''prints issues'''
-    print('\n')
-    print('Repo: ' + issue['repository_url'])
-    print('ID: ' + str(issue['id']))
-    print('Title: ' + issue['title'])
-    print_labels(issue)
+
+    labels = ', '.join([label['name'] for label in issue.get('labels', [])])
```

Comment:
Instead of a new function, I just joined the labels to a single, comma-delimited string with a list comprehension. This is the only actual functionality change, as it modifies the output from multiple "Label:" lines to a single one.